projects/imminence/docker-compose.yml

```diff
@@ -17,28 +18,39 @@ x-imminence: &imminence
   working_dir: /govuk/imminence
 
 services:
+  imminence-pg:
```

Comment:
Does this need to be imminence specific - could it be postgres-14-postgis ? and be with the other db services?

Reply:
It could probably be with the others, I'll move it and see what falls out.